config/database.yml

```diff
@@ -26,4 +26,4 @@ production:
   password: 
 
 cucumber:
-  <<: *test
+<<: *test
```

Comment:
Sorry if this seems nit picking @karenfreemansmith, .yml files are very strict with their indentation rules.